src/transform.ts

```diff
@@ -116,9 +147,9 @@ function transformJSONSchemaToFakerCode(jsonSchema?: OpenAPIV3.SchemaObject, key
           .join(',\n')}
     }`;
     case 'array':
-      return `[...(new Array(faker.datatype.number({ min: ${jsonSchema.minLength ?? 1}, max: ${
-        jsonSchema.maxLength ?? 'MAX_ARRAY_LENGTH'
-      } }))).keys()].map(_ => (${transformJSONSchemaToFakerCode(jsonSchema.items as OpenAPIV3.SchemaObject)}))`;
+      return `[...(new Array(faker.datatype.number({ min: ${jsonSchema.minItems ?? 1}, max: ${jsonSchema.maxItems ?? 'MAX_ARRAY_LENGTH'} }))).keys()].map(_ => (${transformJSONSchemaToFakerCode(
```

Comment:
Array length limits should be specified with `minItems` and `maxItems`. `*length` properties only apply to [strings](https://datatracker.ietf.org/doc/html/draft-wright-json-schema-validation-00#section-4.1).